lib/engine/part/path.rb

```diff
@@ -62,13 +62,29 @@ def initialize(a, b, terminal: nil, lanes: LANES, track: :broad, ignore: nil)
         separate_parts
       end
 
+      def ignore_gauge_compare
```

Comment:
no, we shouldn't use class variables

Reply:
What's the alternative? Instance variables were not working - too hard to find all Path instances being used.